### ISSUE TITLE
update existing Object with write permission on Role not working

### ISSUE THREAD

**Issue by steven274** [ParsePlatform/parse-server#1800]
I have Admin Role with root user.
My object have public read permission and Admin role read write permission.

```
{
    "_id": "WDqygIJ28J",
    "_rperm": [
        "*"
    ],
    "_wperm": [
        "role:Admin"
    ],
    "_updated_at": {
        "$date": "2016-05-16T04:38:42.064Z"
    },
    "_created_at": {
        "$date": "2016-05-16T04:32:56.967Z"
    }
}
```

when I logged in root user, modify the object field and save, it says 'Permission denied for this action'

```
obj.set(request.params.name, request.params.value);
obj.save();
```

**Comment by dcdspace** [comment]
Did you ever solve this?

**Comment by drew-gross** [comment]
This should be fixed by #1989
